### PR TITLE
Lita, Getaround, Samboat, Click2Buy, and https

### PR DESCRIPTION
**Pull request by sunny**
- Add Lita
- Rename Drivy to Getaround
- Rename Clic2Buy to Click2Buy
- Rename Boaterfly to Samboat
- Rename KissKissBankBank to KissKissBankBank & Co
- Use canonical (mostly https) URLs when URL was different

### CODE REVIEW

File: frenchrubyshops.csv [hunk 1]
```diff
@@ -1,99 +1,99 @@
...
-Boaterfly,http://www.boaterfly.com/,product,Paris
-Brand and Celebrities,http://www.brandandcelebrities.com,product,Paris
+Augment,https://www.augment.com/,product,Paris
+Brand and Celebrities,https://www.brandandcelebrities.com/,product,Paris
 Capsens,https://agence.capsens.eu/,product,Paris
 Captain Contrat,https://www.captaincontrat.com/,product,Paris
 Captive,https://captive.fr/,agency,Paris
 Cheerz,https://www.cheerz.com/,product,Paris
-Clic2Buy,http://www.clic2buy.com/,product,Lille
+Clic2Buy,https://www.click2buy.com/,product,Lille
```
Comment (sunny) [inline frenchrubyshops.csv]:
```suggestion
Click2Buy,https://www.click2buy.com/,product,Lille
```

Comment (sunny) [inline frenchrubyshops.csv]:
```suggestion
Click2Buy,https://www.click2buy.com/,product,Lille
```

File: frenchrubyshops.csv [hunk 2]
```diff
@@ -1,99 +1,99 @@
...
-JobTeaser.com,http://www.jobteaser.com/fr,product,Paris
+Je Rêve d'une Maison,https://www.jerevedunemaison.com/,product,Paris
+JobTeaser.com,https://www.jobteaser.com/fr,product,Paris
 Kactus,https://www.kactus.com/,product,Paris
 Kerala Ventures,https://www.kerala.vc/,product,Paris
-Key-Live,http://www.keley-live.com/,agency,Paris
+Key-Live,https://www.keley-live.com/,agency,Paris
 Keycoopt,https://www.keycoopt.com/,product,Lille
-KissKissBankBank,http://www.kisskissbankbank.com/,product,Paris
+KissKissBankBank,https://www.kisskissbankbank.com/,product,Paris
```
Comment (sunny) [inline frenchrubyshops.csv]:
```suggestion
KissKissBankBank & Co,https://www.kisskissbankbank.com/,product,Paris
```

**Review by cveneziani — APPROVED** [review]
Thanks! :confetti_ball:

**Pull request merged**